shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/inmemory/PrivateMetricMetaData.java

```diff
@@ -0,0 +1,45 @@
+package gov.hhs.cdc.trustedintermediary.external.inmemory;
+
+import gov.hhs.cdc.trustedintermediary.metadata.MetaDataEntry;
+import gov.hhs.cdc.trustedintermediary.metadata.MetaDataStep;
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger;
+import gov.hhs.cdc.trustedintermediary.wrappers.MetricMetaData;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import javax.inject.Inject;
+
+/**
+ * Implementation of a class that can be used throughout the project to collect a list of metadata
+ * events
+ */
+public class PrivateMetricMetaData implements MetricMetaData {
```

Comment:
I wonder if this should be renamed to `LoggingMetricMetaData`?  This concrete implementation isn't about private metrics, but about logging the metric metadata.  Thoughts?

Reply:
Are there metrics here? It seems that we're only logging the steps in the process? I'm wondering how is this different from using the logger to record the steps with timestamps and bundleid? I'm sure I'm missing some context here

Reply:
@basiliskus you make a great point! The idea for this initial story was to get the smallest chunk of metadata possible.  So we are recording the step and the bundle id.  In the future we have eyes towards adding other metadata to this so that for each step we possibly have lots of information.  We also need a way to eventually persist this information across restarts. But how we do that is TBD

Reply:
Got it, makes sense. Thanks!

Reply:
@halprin done

Reply:
done.